### PR TITLE
Adds security warning on secrets in containers

### LINKED ISSUE #126: Mention not committing secrets to repo when discussing `Including personal scripts and data in a container`
Committing secrets to GitHub or other repositories is a common error and can be a huge security risk with potentially large real financial costs if in a public cloud context. This should be mentioned in the [05b-advanced-containers](https://github.com/carpentries-incubator/docker-introduction/blob/gh-pages/_episodes/05b-managing-containers.md) episode in the "Including personal scripts and data in a container" section.

Something like.

WARNING: Login credentials including passwords, tokens, secure access tokens or other secrets must **never** be stored in a container.  If secrets are stored, deleting the files in git **does not remove them** from the repository and they will be found and exploited if they are currently, or in the distant future, make public.

I can make a PR if desired.

### PR DESCRIPTION
**Pull request by aturner-epcc**
Closes #126

### CODE REVIEW

File: _episodes/advanced-containers.md [hunk 1]
```diff
@@ -275,6 +275,12 @@ did in the previous section. You also want to think carefully about size -- if y
 run `docker image ls` you'll see the size of each image all the way on the right of
 the screen. The bigger your image becomes, the harder it will be to easily download.
 
+> ## Security warning
+> Login credentials including passwords, tokens, secure access tokens or other secrets
+> must never be stored in a container. If secrets are stored, they will be found and
+> exploited when made public - either in the near or distant future.
```
Comment (sstevens2) [inline _episodes/advanced-containers.md:281]:
```suggestion
> must never be stored in a container. If secrets are stored, they are at high risk to be found and
> exploited when made public.
```
Small suggestion for a language edit but I'm okay with either.

**Review by sstevens2 — APPROVED** [review]
Small suggestion you can take or leave @aturner-epcc.  I think this is a good solution to the issue.  Please merge when you are happy with it.

**Review by jcohen02 — APPROVED** [review]
This looks good to me too, definitely important to point this potential issue out.

I agree that this is fine and can be merged as is but if you are going to make an edit in relation to the language as suggested, an alternative phrasing could be "_...If secrets are stored, they are at high risk of being found and exploited..._"?

File: _episodes/advanced-containers.md (hunk 1 above)
Comment (aturner-epcc) [reply _episodes/advanced-containers.md]: Done. Thanks for the suggestion.

**Pull request merged**